charts/prometheus-kafka-exporter/templates/deployment.yaml

```
@@ -63,6 +63,9 @@ spec:
           {{- if not .Values.sasl.handshake }}
             - '--sasl.handshake=false'
           {{- end }}
+          {{- if .Values.sasl.awsiam.enabled }}
+            - '--sasl.mechanism={{ .Values.sasl.awsiam.mechanism }}'
+          {{- end }}
```

With support for aws-region:

```suggestion
          {{- if .Values.sasl.awsiam.enabled }}
            - '--sasl.mechanism={{ .Values.sasl.awsiam.mechanism }}'
            {{- if .Values.sasl.awsiam.awsRegion }}
            - '--sasl.aws-region={{ .Values.sasl.awsiam.awsRegion }} 
            {{- end }}
          {{- end }}
```